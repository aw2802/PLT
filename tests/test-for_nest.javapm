public class TestFor_Nest{
  public void main(){
    int i;
    int j;
    for (i = 0; i < 3; i = i+1){
      for(j = 0; j < 2; j = j+1){
        println("Inside");
      }
    }
  }
}
