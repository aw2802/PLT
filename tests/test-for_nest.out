Inside
Inside
Inside
Inside
Inside
Inside
